ignite/handlers/pytorch_profiler.py

```diff
@@ -0,0 +1,144 @@
+# coding: utf-8
+import datetime
+import os
+from typing import Any, Callable, Union
+
+import torch
+
+import ignite.distributed as idist
+from ignite.engine import Engine, Events
+
+
+class PyTorchProfiler:
+    """PyTorch Profiler for performance debugging.
+
+    The PyTorch profiler is a tool that collects both GPU hardware and PyTorch related
+    information, correlates them, performs automatic detection of bottlenecks in the model,
+    and generates recommendations on how to resolve these bottlenecks.
+
+    Examples:
+        .. code-block:: python
+
+            from ignite.handlers import PyTorchProfiler
```

Comment:
The import is missing in `engine.handlers.__init__.py`

Reply:
Added this